Gemfile

```diff
@@ -4,7 +4,6 @@ gemspec
 
 group :test do
   gem 'codeclimate-test-reporter', '~> 1.0.0'
-  gem 'rubocop',                   '~> 0.47'
```

Comment:
Nice. Weird that we have different requirements. I bet we've had some incompatibilities in the real released gem due to this.